Add param for build_push step in build_push_image job (#26)
ricardo-ch/ricardo-orbs: src/jobs/README.md

...
```

> ### Build & Quality Gate Job

Reviewer: ```suggestion
### Build & Quality Gate Job for Go Applications
```

> **Name**: build_quality_gate

Reviewer: ```suggestion
**Name**: go_build_test
```

**Parameters**:

- **executor**, [circleci executor](https://circleci.com/docs/2.0/configuration-reference/#docker--machine--macos--windows-executor)
- **additional_tests** is a list of additional steps to run, e.g. run tests on common. The default is: empty list
- **work_dir** working directory for executing build and test commands

> Job runs install_build_test step, runs additional steps if there are any, and persist resulting `app` filet to workspace (to be used by following jobs) 

Reviewer: ```suggestion
Job runs install_build_test step, runs additional steps if there are any, and persist resulting `app` file to workspace (to be used by following jobs) 
```

**Examples**

In this example:

- user defined executor is used
- custom steps

```yaml
...
jobs:
...

> - ric-orb/build_quality_gate_job:

Reviewer: ```suggestion
- ric-orb/go_build_test:
```

    executor:
      name: go/default
      tag: '1.17'
    context: dev
    additional_tests:
      - run:
          name: Running tests on common
          command: make test
          working_directory: common
...
```

> In this example is default setting, only required is added:

Reviewer: ```suggestion
This example uses default settings and only the minimum required configuration is added:
```
